matzip-app-external-api/src/test/java/com/woowacourse/matzip/application/RestaurantServiceTest.java

```diff
@@ -332,4 +337,23 @@ class RestaurantServiceTest {
                 .extracting(RestaurantTitleResponse::getReviewCount)
                 .containsExactlyInAnyOrder(4, 3);
     }
+
+    @Test
+    void 식당_목록_조회시_북마크수도_조회한다() {
+        Restaurant restaurant = restaurantRepository.save(createTestRestaurant(1L, 1L, "테스트식당", "테스트주소"));
+
+        for (int i = 1; i <= 3; i++) {
+            Member member = createTestMember("githubId" + i);
+            memberRepository.save(member);
+
+            bookmarkRepository.save(createTestBookmark(member, restaurant));
+        }
```

Comment:
```suggestion
        Member member1 = memberRepository.save(createTestMember("githubId1"));
        bookmarkRepository.save(createTestBookmark(member1, restaurant));
        Member member2 = memberRepository.save(createTestMember("githubId2"));
        bookmarkRepository.save(createTestBookmark(member2, restaurant));
        Member member3 = memberRepository.save(createTestMember("githubId3"));
        bookmarkRepository.save(createTestBookmark(member3, restaurant));
```

가능하면 테스트에는 for와 같은 로직보다는 차라리 하드코딩해서 각 값을 넣어주는 것이 좋을 것 같아요.
물론 로직이 단순할 수도 있지만 1~3이라는 로직자체가들어갔다는 것은 그 구문을 이해해야하는 범위가 늘어나 직관적이지 못하고 간단하더라도 오류를 범할 수 있기 때문이에요.

Reply:
호오 확실히 직관적으로 보이네요!

Reply:
3개 정도는 하드코딩으로 넣는 것이 훨씬 직관적으로 보이네요!